ivadomed/training.py

```diff
@@ -263,10 +282,15 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
 
             # UPDATE BEST RESULTS
             if val_loss_total_avg < best_validation_loss:
+                # Save checkpoint
+                state = {'epoch': epoch + 1, 'state_dict': model.state_dict(),
+                         'optimizer': optimizer.state_dict(), 'tensorboard_logger': writer,
```

Comment:
I am having a mistake here: `TypeError: cannot serialize '_io.BufferedWriter' object` It seems to be due to saving the writer. There is probably a work around this, I'll look into it :)